test/python/algorithms/test_measure_error_mitigation.py

```diff
@@ -332,7 +332,7 @@ def test_measurement_error_mitigation_with_vqe_ignis(self, config):
 
     @unittest.skipUnless(HAS_AER, "qiskit-aer is required for this test")
     @unittest.skipUnless(HAS_IGNIS, "qiskit-ignis is required to run this test")
-    def test_callibration_results(self):
+    def test_calibration(self):
```

Comment:
Does this test cover the new changes? If yes, why? 🙂

Reply:
I added a transpiled circuit to it to test the change in Quantum Instance. Since I fixed the method name spelling, I made it a shorter name.

Reply:
Now this method went back to the way it used to be and a new unit test method was created for this PR.